hyungjoonYoon/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "myapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^1.0.3",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql": "^2.18.1",
+    "typeorm": "^0.3.10"
+  }
```

dependecies와 devDependencies의 차이점에 대해 학습해주세요!

배포 단계에 반드시 필요한 패키지/모듈이 있는 반면, 개발 단계에서만 필요한 패키지/모듈이 존재합니다.

이 둘의 차이점을 명확히 알고 아래의 명령어를 통해 수정해주세요!

npm install "package-name"
npm install --save-dev "package-name"
npm uninstall "package-name"

```suggestion
  "dependencies": {
    "cors": "^2.8.5",
    "dotenv": "^16.0.3",
    "express": "^4.18.2",
    "mysql": "^2.18.1",
    "typeorm": "^0.3.10"
  },
  devDependencies: {
      "morgan": "^1.10.0",
      "nodemon": "^3.0,0"
  }
```